CommonControls/FileTypes/RigidModel/Vertex/Formats/StaticVertex.cs

```diff
@@ -42,6 +43,39 @@ public CommonVertex Read(RmvVersionEnum rmvVersion, byte[] buffer, int offset, i
             return vertex;
         }
 
+        static Vector3 _swapXZ(Vector3 v)
+        {
+            return new Vector3(v.Z, v.Y, v.X);
+        }
+        public CommonVertex Read(RmvVersionEnum rmvVersion, byte[] buffer, int offset, int vertexSize)
+        {
+            var vertexData = ByteHelper.ByteArrayToStructure<Data>(buffer, offset);
+
+            var vertex = new CommonVertex()
+            {
+             
+                Position = VertexLoadHelper.CreatVector4HalfFloat(vertexData.position).ToVector4(1),
+
+                /*  VertexFormat = ´default` format has X and Z swapped 
+                    'bitangent' is stored before 'tangent' when VertexFormat = ´default` 
+                */
+                Normal = _swapXZ(VertexLoadHelper.CreatVector4Byte(vertexData.normal).ToVector3()),
+                BiNormal = _swapXZ(VertexLoadHelper.CreatVector4Byte(vertexData.tangent).ToVector3()),
```

Comment:
One can swap the tanget and binormal in the vertexData object to make it less confusing

Reply:
You want code below this object initilization, that does the swap instead? Or?